src/main/java/de/tum/in/www1/artemis/web/rest/admin/AdminSystemNotificationResource.java

```diff
@@ -0,0 +1,102 @@
+package de.tum.in.www1.artemis.web.rest.admin;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import de.tum.in.www1.artemis.domain.notification.Notification;
+import de.tum.in.www1.artemis.domain.notification.SystemNotification;
+import de.tum.in.www1.artemis.repository.SystemNotificationRepository;
+import de.tum.in.www1.artemis.security.annotations.EnforceAdmin;
+import de.tum.in.www1.artemis.service.SystemNotificationService;
+import de.tum.in.www1.artemis.web.rest.SystemNotificationResource;
+import de.tum.in.www1.artemis.web.rest.errors.BadRequestAlertException;
+import de.tum.in.www1.artemis.web.rest.util.HeaderUtil;
+
+/**
+ * REST controller for administrating system notifications.
+ */
+@RestController
+@RequestMapping("api/admin/")
+public class AdminSystemNotificationResource {
+
+    private final Logger log = LoggerFactory.getLogger(SystemNotificationResource.class);
```

Comment:
❌ Required Change: The logger seems to refer to the wrong class. Please check the other occurrences also